src/frontend/src/pages/ProjectsPage/ProjectsTable.tsx

```diff
@@ -168,7 +222,7 @@ const ProjectsTable: React.FC = () => {
           }
         }}
         onFilterModelChange={(filterModel: GridFilterModel) => {
-          localStorage.setItem('projectsTableFilter', JSON.stringify(filterModel.items[0]));
+          localStorage.setItem('projectsTableFilter', JSON.stringify(filterModel.items[0] ?? null));
```

Comment:
getting the same problem but for null, I think we'll need to do a ternary and not set the localStorage if the filterModal.items[0] is undefined or null:
![image](https://github.com/Northeastern-Electric-Racing/FinishLine/assets/29521172/cfe42993-1a71-4534-b204-558c7504205e)